Ports/python3/package.sh

```diff
@@ -2,6 +2,8 @@
 
 source version.sh
 
+PATH="${SERENITY_SOURCE_DIR}/Toolchain/Local/python/bin:${PATH}"
```

Comment:
Should this not be `export`ed?

Reply:
good point. but, it worked without locally. I've exported for consistency. same for ruby.

Reply:
Interesting, I'd assume that the change doesn't actually reach the python build in that case, since its not exported.